### PR TITLE
Faq section

### PR DESCRIPTION
**Pull request by Ma9da**
image

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -104,4 +104,93 @@ flex-wrap: wrap;
     background-repeat: no-repeat;
     background-size: cover;
 }
-/***** End hero *****/
\ No newline at end of file
+/***** End hero *****/
+/***** faq *****/
+.reviews{
+    background: var(--light-gray);
+    padding: 50px 0;
```
Comment (Safei-Ashraf) [inline css/style.css]: rem

File: css/style.css [hunk 2]
```diff
@@ -104,4 +104,93 @@ flex-wrap: wrap;
...
+.faq__cards-card-header{
+    margin: 0 10px;
+}
+.faq__cards-card-body h4{
+font-weight: 700;
+text-align: left;
+color: var(--text-color);
+}
+/***** End faq *****/
+@media screen and (max-width: 1200px) and (min-width:577px){
```
Comment (Safei-Ashraf) [inline css/style.css]: you don't need to set min and max for break point, use only value, will help avoid rules duplication on  other breakpoints as well.

File: index.html [hunk 3]
```diff
@@ -56,6 +56,209 @@ <h3><a href="#">physionic</a></h3>
...
+                    FAQ
+                </h3>
+                <p class="section__desc">
+                    Problems trying to resolve the conflict between
+                </p>
+                <p class="section__desc">
+                    the two major realms of Classical physics: Newtonian mechanics
+                </p>
+            </div>
+            <div class="faq__cards reviews__cards">
```
Comment (Safei-Ashraf) [inline index.html]: use semantic HTML elements e.g <section>

File: index.html [hunk 4]
```diff
@@ -56,6 +56,209 @@ <h3><a href="#">physionic</a></h3>
...
+                            Things on a very small scale
+                            behave like nothing
+                        </p>
+                    </div>
+                </div>
+                <!-- card end  -->
+                <!-- card start  -->
+                <div class="faq__cards-card">
+                    <div class="faq__cards-card-header">
+                        <svg class="arrow-icon" viewBox="0 0 9 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Safei-Ashraf) [inline index.html]: img instead of svg

**Review by Safei-Ashraf — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 2 above)
Comment (Ma9da) [reply css/style.css]: the mobile size sometimes have completely different icons/images

Comment (Safei-Ashraf) [reply css/style.css]: yes that's a regular use case, that is totally doable with the usual approach (one screen size), even if you need to override it in your default styling, or a another media query

**Pull request merged**